### PR TITLE
feat: replace incorrect chapter of Admiral's Ward

### LINKED ISSUE #74: The Admiral's Ward (1883) has two different London editions
Multivolume edition (51 chapters):
- Vol 1: https://archive.org/details/admiralsward01alex/page/n4
- Vol 2: https://books.google.com/books?id=EfEBAAAAQAA
- Vol 3: https://archive.org/details/admiralsward03alex/page/n4

Popular edition (50 chapters):
- https://books.google.com/books?id=9j82vwEACAAJ

Both published in London by Bentley

### PR DESCRIPTION
**Pull request by riddella**
Closes #74

### CODE REVIEW

**Comment by riddella** [comment]
@ayarnell everything look OK? I entered this one.

File: texts/1053__alexander_admirals_ward_chp49of51_seed1053.html [hunk 1]
```diff
@@ -0,0 +1,614 @@
...
+whatever happened.’
+</p>
+
+<p>
+‘ Oh, Mrs. Crewe !’ interrupted Laura in a tone
+of genuine horror, for the suggestion terrified her.
+</p>
+
+<p>
+‘ My dear Mrs. Crewe/ said the Admiral, with
```
Comment (ayarnell) [inline texts/1053__alexander_admirals_ward_chp49of51_seed1053.html]: Looks like an accidental / in line 244. Should be ' My dear Mrs. Crewe,'

File: texts/1053__alexander_admirals_ward_chp49of51_seed1053.html [hunk 2]
```diff
@@ -0,0 +1,614 @@
...
+farthing ! Well ; it’s not every heiress that knows
+she was chosen for herself alone, as you know my
+dear blessed boy chose you ; and when I say, Laura
+Piers, you deserve him,’ added the mother, with a
+little sob, ‘ I give you the highest praise you will
+ever get, be the other what it may.’
+</p>
+
+<p>
+‘ I think you do/ said Laura, drawing nearer
```
Comment (ayarnell) [inline texts/1053__alexander_admirals_ward_chp49of51_seed1053.html]: Same issue here as in ln 244, accidental /

**Comment by riddella** [comment]
Thanks for sharp eyes @ayarnell .

**Pull request merged**